cpp/src/parquet/column_writer.cc

```diff
@@ -2163,6 +2183,9 @@ Status TypedColumnWriterImpl<DoubleType>::WriteArrowDense(
 
 // ----------------------------------------------------------------------
 // Write Arrow to BYTE_ARRAY
+template <>
+void TypedColumnWriterImpl<ByteArrayType>::UpdateBloomFilterArray(
+    const ::arrow::Array& values);
```

Comment:
Why redeclare this method? It's already redeclared above.

Reply:
```
arrow/cpp/src/parquet/column_writer.cc:2423:44: error: explicit specialization of 'UpdateBloomFilterArray' after instantiation
void TypedColumnWriterImpl<ByteArrayType>::UpdateBloomFilterArray(
```

Otherwise this would be raised.